templates/zerver/app/index.html

```diff
@@ -37,9 +37,20 @@
     #app-loading-middle-content h3 {
     text-align: center;
     }
+    @keyframes showAtEnd {
+    99% {
+    visibility: hidden;
+    }
+    100% {
+    visibility: visible;
+    }
+    }
     #app-loading-bottom-content {
     top: unset;
     bottom: 20px;
+    animation: 7s showAtEnd;
```

Comment:
Change this value of `7s` to the value you want to test with this setting (`disable cache` and `Fast 3G`) in your Chrome developer network tab.

<img width="613" alt="Screenshot 2022-06-06 at 2 21 02 PM" src="https://user-images.githubusercontent.com/25124304/172129059-76bf512a-58e3-47da-a491-1986af5b6d8f.png">